src/applications/facility-locator/tests/components/UrgentCareSearch.unit.spec.js

```diff
@@ -1,8 +1,10 @@
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
-import { urgentCareCall } from '../../components/SearchResult';
+import {
+  urgentCareCall,
+  posProviderName,
+} from '../../components/search-results-items/UrgentCareResult';
```

Comment:
I'd suggest we move the tests for `urgentCareCall` and `posProviderName`  into an `UrgentCareResult` unit test.